config/dispatcher/dispatcher.ex

```diff
@@ -1,134 +1,168 @@
 defmodule Dispatcher do
-  use Plug.Router
-
-  def start(_argv) do
-    port = 80
-    IO.puts "Starting Plug with Cowboy on port #{port}"
-    Plug.Adapters.Cowboy.http __MODULE__, [], port: port
-    :timer.sleep(:infinity)
-  end
-
-  plug Plug.Logger
-  plug :match
-  plug :dispatch
-
-  # In order to forward the 'themes' resource to the
-  # resource service, use the following forward rule.
-  #
-  # docker-compose stop; docker-compose rm; docker-compose up
-  # after altering this file.
-  #
-  # match "/themes/*path" do
-  #   Proxy.forward conn, path, "http://resource/themes/"
+  use Matcher
+
+  define_accept_types [
+    json: [ "application/json", "application/vnd.api+json" ],
+    html: [ "text/html", "application/xhtml+html" ],
+    any: [ "*/*" ]
+  ]
+
+  define_layers [ :static, :web_page, :api_services, :not_found ]
+
+  @html %{ accept: %{ html: true } }
+  @json %{ layer: :api_services, accept: %{ json: true } }
+  @any %{ accept: %{ any: true } }
+
+  ###############
+  # STATIC
+  ###############
+
+  # match "/*_path", @html do
+  #   # *_path allows a path to be supplied, but will not yield
+  #   # an error that we don't use the path variable.
+  #   forward conn, [], "http://mandaten/index.html"
   # end
 
-  options _ do
-    send_resp( conn, 200, "Option calls are accepted by default." )
+  # match "/assets/*path", @any do
+  #   forward conn, path, "http://mandaten/assets/"
+  # end
+
+  # get "/assets/*path", %{ layer: :static } do
+  #   forward conn, path, "http://mandaten/assets/"
+  # end
+
+  # get "/favicon.ico", %{ layer: :static } do
+  #   send_resp( conn, 404, "" )
+  # end
```

Comment:
These are probably needed, but locally it didn't seem to matter :/ I don't understand how any of this was working. Things like `/@appuniversum/ember-appuniversum/fonts/blabla` were also resolving, even though they don't match any of the matchers I think?